.github/workflows/toolchain.yml

```diff
@@ -0,0 +1,32 @@
+name: Toolchain
+
+on: [push, pull_request]
+
+jobs:
+  toolchain:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        tc: [kindle, kindle5, kindlepw2, kobo, nickel, remarkable, cervantes, pocketbook, bookeen]
```

Comment:
Incidentally, there are also a few additional Kobo-TCs. I'm not really sure why we distinguish so much on Kindle though tbh; does it actually matter that much in practice?

Reply:
> few additional Kobo-TC

? I took this list straight from `gen-tc`

> I'm not really sure why we distinguish so much on Kindle though tbh; does it actually matter that much in practice?

@NiLuJe will actually know, but i think there are different kernel versions and also cortex-a8 vs a9

Reply:
There's a kobomk7 and a kobomk8, apparently not listed in gen-tc. ;-P